packages/flutter_tools/lib/src/localizations/gen_l10n.dart

```diff
@@ -1150,9 +1150,7 @@ class LocalizationsGenerator {
   // itself does not generate the output files.
   String _generateCode() {
     bool isBaseClassLocale(LocaleInfo locale, String language) {
-      return locale.languageCode == language
-          && locale.countryCode == null
-          && locale.scriptCode == null;
+      return locale.languageCode == language;
```

Comment:
@HansMuller 
Languages with script code also needed a base locale file so which we won't anymore. This passes all Flutter Gallery tests locally.

Reply:
This method doesn't appear to mean that its locale is the "base" locale anymore.

What does this line, from `getLocalesForLanguage()` mean now?
```dart
          return !isBaseClassLocale(locale, language) && locale.languageCode == language;
```

Reply:
That's a good point, that was an oversight from me.  I updated the code just check for locale language